crates/typst-library/src/compute/calc.rs

```diff
@@ -443,11 +443,13 @@ pub fn fact(
     Ok(fact_impl(1, number).ok_or("the result is too large")?)
 }
 
-/// Calculates a permutation.
+/// Calculates the `k`-permutation of `n`.
 ///
 /// ```example
-/// #calc.perm(10, 5)
-/// ```
+/// $
+/// "perm"(n, k) &= n!/((n - k)!) \
+/// "perm"(5, 3) &= #calc.perm(5, 3)
+/// $
```

Comment:
The current equation style in the docs is 
```typst
$ "perm"(n, k) &= n!/((n - k)!) \
  "perm"(5, 3) &= #calc.perm(5, 3) $
```
I am aware that few people use this style, but I would like to keep it consistent for now and decide at a later point whether we want to change it.